vue.config.js

```diff
@@ -35,6 +43,42 @@ module.exports = {
         res.writeHead(200, { 'Content-Type': 'application/json' });
         res.end(JSON.stringify(result));
       });
+
+      // Mock call to record start, with this format
+      // {uuid: ..., queue_id: ..., status: "in_progress", status_description: ..., description: ...}
+      app.all('/calltorecord/start', (req, res) => {
+        const result = {
+          uuid: `${Math.random().toString(36).substr(2, 16)}.${Math.random().toString(36).substr(2, 10)}`,
+          queue_id: Math.floor(Math.random() * (1000 + 1)),
+          status: 'in_progress',
+          status_description: '',
+          description: 'Test call-to-record audio',
+        }
+        res.cookie(result.uuid, 'in_progress')
```

Comment:
@bzabos , this is the new alternative with cookies. I
FYI: had to add `cookie-parser` package to be able using `res.cookie`